meetings/2019-03-f2f.md

```diff
@@ -0,0 +1,247 @@
+  * Developers don't want multiple pipelines for their content. They have been incentivised by being at the top of search results.
+  * Would like to see a decoupling of AMP and Search results / News.
+
+### Meeting with AMP Letter Signatories
+
+* **AI(@tobie): get guests to validate minutes before making them public. **
+
+**[PLACEHOLDER FOR AMP LETTER SIGNATORIES MEETING MINUTES]**
+
+## Day 2
+
+#### Attendees:
+
+* Terence Eden @edent
+* Tobie Langel @tobie
+* Charles Vazac @cvazac
+* Joe Alicata @wirelessjoe
+* Léonie Watson @ljwatson
+
+#### Agenda:
+
+* ad-tech primer -> define what the AC's role would be in that space
+* organise verticals in groups (AC/TSC dependant?) [issue #5][5] [issue #15][15]
+* foundation update
+* meeting around outreach w/ Ben Morss
+* accessibility + [issue #5][5]
+* AMP for email
+* Document AMP architecture + update standard list
+
+### Foundation
+
+* Tobie gives overview
+* Who is the target membership?
+* How are they going to be contacted? Outreach process.
+  * Outreach could be done by the foundation itself
+  * Ask AC/TSC membership would they be interested in joining the foundation
+  * At what price point?
+  * Whether we want the foundation limited to just legal/accounting or include outreach/etc.
+* Foundation should act as a steward for content when / if AMP caches are turned off.
+
+### Ad-tech primer / revenue
+
+* Joe gives overview
+* containerized in AMP (sandbox)
+* system:
+  * good doc on the topic in the AMP doc
+  * container in the page
+  * add server
+  * inside the code that comes back from the ad-server
+* As a result: limitations
+  * creative
+  * data (lower access to targeting data)
+* "Auction"-based ad-style weren't supported at first
+* https://www.ampproject.org/docs/ads/introduction_ads
+* Add-network's role: supply/demand aggregator 
+* Creative limitations big burden on publishers
+* built on safe-frame
+* benefit from a user's perspective: speed
+* AMP HTML Ads => AMP in the ad itself, served on any kind of Web page
+* Who guarantees the AMP format validity in that case?
+
+### Accessibility
+
+See [issue #5][5].
+
+* Many of the components are not sufficiently accessible.
+* Accessibility tree - usually used by browsers - they understand what ever HTML element does.
+  * AMP doesn't rely on native elements enough
+* Media Player is a real concern. Controls show up as list boxes rather than buttons
+* Same kinds of issues for the carousel
+* Some bits are good - like collapsing
+* Is there a process for accessibility checking before a component is added?
+  * No - but there needs to be
+  * The AC would like to see a process in place by 2020-01-01
+* The AC would like all components to meet WCAG 2.1 before release
+* By the AMP Contributor Summit - produce a roadmap and timetable for raising the existing components up to the expected standard
+* **AI(): reach out to TSC/Accessibility WG and ask for audit of existing components.**
```

Comment:
**AI(): reach out to TSC/Accessibility WG and ask for an audit of existing components.**